### PR TITLE
Implemented presentor and views for bootstrap and md

### PR DESCRIPTION
**Pull request by bpasha7**

### CODE REVIEW

**Review by trueromanus — CHANGES_REQUESTED** [review]
Need few changes

File: src/views/bootstrap/ListGroupView.vue [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<template>
```
[the remark this answers is not in the document]
Comment (bpasha7) [reply src/views/bootstrap/ListGroupView.vue:1]: renamed to ListBox

File: src/examples/helloworld/App.vue [hunk 2]
```diff
@@ -51,31 +51,76 @@
...
+        <div class="container">
+            <div class="stack">
+                <h3>Bootstrap</h3>
+                <list-group-state v-model="listItems"
+                    :active-handler="listActiveItem"
+                    @clicked="listItemClicked($event)">
+                    <template #default="{ context }">
+                        <list-group-view :context="context">
+                            <template #content="{ item }">
+                                {{item}}
```
[the remark this answers is not in the document]
Comment (bpasha7) [reply src/examples/helloworld/App.vue:63]: done=)

File: src/states/ListGroupState.vue [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+            default: () => false
+        }
+    },
+    methods: {
+        click($event) {
+            if (this.disable) return;
+
+            this.$emit(`clicked`, $event);
+        }       
+    }
```
[the remark this answers is not in the document]
Comment (bpasha7) [reply src/states/ListGroupState.vue:31]: We have decided to implement ListBox logic

**Review by trueromanus — APPROVED** [review]
done

**Pull request merged**